desktop/cmp/dash/canvas/DashCanvasModel.js

```diff
@@ -64,7 +64,7 @@ export class DashCanvasModel extends HoistModel {
     //------------------------
     // Implementation properties
     //------------------------
-    /** @member {DOMElement} */
+    /** @member {React.DOMElement} */
```

Comment:
Pretty sure this was incorrect before - since the `ref` is a Ref, and to get to the DOMElement you need to do a `ref.current` - I think the correct type would be `RefObject<DOMElement>`

Reply:
Definitely agree- was incorrect before. JSDocs autosuggested React.DOMElement, but I'm sure you're right with RefObject<DOMElement> being the proper type